### PR TITLE
feat: Add import/extensions rule

### PR DESCRIPTION
**Pull request by dkimura**
## TL;DR

- Add import/extensions rules
  - ref: https://github.com/benmosher/eslint-plugin-import/issues/1615

## Check list

- Pass CI

### CODE REVIEW

File: packages/eslint-config-globis/shared.js [hunk 1]
```diff
@@ -9,6 +9,18 @@ module.exports = {
...
+      'error',
+      'ignorePackages',
+      {
+        js: 'never',
+        jsx: 'never',
+        mjs: 'never',
+        ts: 'never',
+        tsx: 'never',
+      },
+    ],
```
Comment (shoota) [inline packages/eslint-config-globis/shared.js:23]: What is difference from `"import/extensions": ["error", "never"]` ?

Comment (dkimura) [reply packages/eslint-config-globis/shared.js:23]: I wanted to set is "do not set the extension of JavaScript files".
Also, I want to enter `.svg` without omitting.

Comment (shoota) [reply packages/eslint-config-globis/shared.js:23]: ah hah 👍

**Review by shoota — APPROVED** [review]
LGTM

**Pull request merged**